helpers/log_context.py

```diff
@@ -58,48 +58,13 @@ def populate_from_sqlalchemy(self, dbsession):
             return
 
         try:
-            can_identify_commit = self.commit_id is not None or (
-                self.commit_sha is not None and self.repo_id is not None
-            )
+            # - commit_id (or commit_sha + repo_id) is enough to get everything else
+            #   - however, this slams the commit table and we rarely really need the
+            #     DB PK for a commit, so we don't do this
+            # - repo_id is enough to get repo and owner
+            # - owner_id is just enough to get owner
 
-            # commit_id or (commit_sha + repo_id) is enough to get everything else
-            if can_identify_commit:
-                query = (
-                    dbsession.query(
-                        Commit.id_,
-                        Commit.commitid,
-                        Repository.repoid,
-                        Repository.name,
-                        Owner.ownerid,
-                        Owner.username,
-                        Owner.service,
-                        Owner.plan,
-                    )
-                    .join(Commit.repository)
-                    .join(Repository.owner)
-                )
-
-                if self.commit_id is not None:
-                    query = query.filter(Commit.id_ == self.commit_id)
-                else:
-                    query = query.filter(
-                        Commit.commitid == self.commit_sha,
-                        Commit.repoid == self.repo_id,
-                    )
-
-                (
-                    self.commit_id,
-                    self.commit_sha,
-                    self.repo_id,
-                    self.repo_name,
-                    self.owner_id,
-                    self.owner_username,
-                    self.owner_service,
-                    self.owner_plan,
-                ) = query.first()
-
-            # repo_id is enough to get repo and owner
-            elif self.repo_id:
+            if self.repo_id:
```

Comment:
So the intent here is to reenables the log context without the commit?

Reply:
yes

the stated problem is that populating the log context slammed the commit table. turns out if we don't query the commit table the only thing we are likely to lose is the DB PK of the commit which we don't tend to use anyway. so this PR stops querying the commit table

this may just move the problem, and we should maybe do the followup described in the PR description anyway